src/platform_impl/macos/view.rs

```diff
@@ -185,6 +190,7 @@ declare_class!(
                 }
 
                 this.state.input_source = this.current_input_source();
+                Ivar::write(&mut this.accepts_first_mouse, accepts_first_mouse);
```

Comment:
Nit: I'd like the variable to be initialized just below the other `Ivar::write` (so one can easily know that calling methods on `this` is sound)